modules/tinymce/src/core/test/ts/browser/lookup/UserLookupTest.ts

```diff
@@ -0,0 +1,157 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Fun } from '@ephox/katamari';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import type Editor from 'tinymce/core/api/Editor';
+import type { User } from 'tinymce/core/lookup/UserLookup';
+
+describe('browser.tinymce.core.UserLookupTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    current_user_id: 'test-user-1',
+    fetch_user_by_id: (userId: string): Promise<User> => {
+      return new Promise((resolve) => {
+        setTimeout(() => {
+          resolve({
+            id: userId,
+            name: 'Test User',
+            avatar: 'test-avatar.png',
+            description: 'Test Description',
+          } as User);
+        }, 100);
```

Comment:
This is a bit high we just need to make sure it's async right so 0 would be fine as well and would make tests run faster.